draft-ietf-scitt-architecture.md

```diff
@@ -683,15 +683,9 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 # Privacy Considerations
 
-Transparency Services are often publicly accessible.
-Issuers should treat Signed Statements (rendering them as Transparent Statements) as publicly accessible.
-In particular, a Signed Statement Envelope and Statement payload should not carry any private information in plaintext.
+Transparency Services may be publicly accessible and Issuers should in those cases treat Signed Statements (rendering them as Transparent Statements) as publicly accessible. In this case, a Signed Statement Envelope and Statement payload should not carry any confidential information in the clear. Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
```

Comment:
Strictly speaking the *highly* immutable / non-removable bit according to the tech stack we've defined is only a hash. There are plausible deniability arguments to be made for even not wanting a hash of embarrassing or incriminating content on an immutable append-only log but I don't think those are in scope of SCITT.

Reply:
Good point. This should be noted in there.

Reply:
I have added text to clarify this aspect.